utils/test/testinput/gdas_rtofsinsitu.yaml

```diff
@@ -0,0 +1,13 @@
+provider: RTOFS
+window begin: 2021-03-24T15:00:00Z
+window end: 2021-03-24T21:00:00Z
+output file: rtofsinsitu_2024032600.profile.nc4

```

Comment:
In my opinion, nc4 were used for test example and all other converters output nc files instead of nc4.

Reply:
That's not correct @apchoiCMD , `.nc4` is the suffix that we've been using so far.